adal/build.gradle

```diff
@@ -42,13 +42,14 @@ android {
     }
 
     lintOptions {
-        abortOnError false
+        abortOnError true
         disable 'MissingPermission'
         disable 'LongLogTag'
         disable 'DefaultLocale'
         disable 'UnusedResources'
         disable 'GradleDependency'
         disable 'GradleOverrides'
+        disable 'OldTargetApi'
```

Comment:
I also don't see a reason why we suppress at where we hit the error. OldTargetApi is a pretty specific suppression if we want to ignore the target version warning.